chapters/2.md

```diff
@@ -1,211 +1,170 @@
+an outline for the module content so that every time the custom element is seen
+in an HTML file the same content will be stamped in place of its name.
+
+TODO The template actually refers to template in transpiler terms. Include here?
+TODO Probably not good to load newbies with that. Should defer to later.
+
+## JavaScript at the Bottom of the Page
+```javascript
+  Polymer({
+    is: 'simple-element',
+    ...
+  });
+```
+Whaaaaaaat? Why is there JavaScript at the bottom of this HTML file? Don't worry.
+That seems a bit odd until you get used to it. The reason this JavaScript is in
+our HTML file is actually simple. In Polymer, what we're specifying in our html
+file is a custom element. However, custom elements that only allow us to specify
+the markup to use in place of that element are extremely limited. What we need is
+a manner to couple that markup with functionality. It's starting to make sense,
+right? The JavaScript is present in the HTML file because what we're defining
+is a custom element not just html. And that custom element has to be capable of
+having custom behavior otherwise there wouldn't be much of a point in having
+custom elements in the first place. We define our custom behavior in that JavaScript.
+
+### The Polymer Function
+You may have noticed that our element definition appears to be the input to a function
+called Polymer. First of all, good eye! The Polymer(...) function allows us to define
+the name of our custom element (this must match the id attribute present in the
+`<dom-module>` element), [element properties](https://www.polymer-project.org/1.0/docs/devguide/properties),
+and more.
+
+### But, Defining Styling, Markup and Logic in the Same File is a Code Smell!
+TODO Should this be included here or no?
+
+## Web Components
+Previously, four aspects of our simple element were identified as being notable
+for polymer in our `simple-element.html` file:
+  1. The link at the top of the page with rel="import".
+  2. The unfamiliar `<dom-module>` tag surrounding all of the page content.
+  3. A `<template>` tag containing styles and other markup.
+  4. Some JavaScript at the bottom of the page.
+
+Well, actually these four items aren't specific to Polymer. They're part of an
+awesome standard coming forth in the web domain called the [Web Components Standard](https://developer.mozilla.org/en-US/docs/Web/Web_Components).
+This standard attempts to increase the modularity and flexability of the web
+platform by introducing four concepts which map well to those we identified earlier:
+  1. HTML Imports Standard.
+  2. Shadow DOM Standard.
+  3. HTML Templates Standard.
+  4. Custom Elements Standard.
+  
+Polymer was created to provide browser compatibility and easy adoption of the
+web component standard. Don't worry if this is fuzzy to you right now because
+you're absorbing a lot of information! We'll go into the nitty gritty of those
+individual standards in separate lessons.
```

Comment:
Is there any way we could sum this chapter up in a picture? sort of like a screenshot of an actual component, with color-coded boxes of your 4 points there

---
imports

Polymer coordinates:

dom module
-> styles (shadow-dom shoutout)
-> template
-> js

Reply:
Love it. I have a written TODO on a desk note pad to walk through this tutorial at the end and take pictures of the whole process. A picture is worth 1000 words. Instead of describing the whole process I want images so that users can easily follow what's happening. Excellent suggestion!
- [ ] Upload images for all appropriate steps